src/interfaces/index.ts

```diff
@@ -0,0 +1,49 @@
+import {ReactNode} from 'react';
+import {
+  GestureResponderEvent,
+  ImageSourcePropType,
+  ImageStyle,
+  StyleProp,
+  TextStyle,
+  ViewStyle,
+} from 'react-native';
+
+export interface TextProps {
```

Comment:
declare the props in the componente like:

interface Props ....

Reply:
all interface inside the file or only TextProps